src/storage-types/src/sources.rs

```diff
@@ -1290,11 +1292,16 @@ mod tests {
         assert!("".parse::<Timeline>().is_err());
     }
 
-    fn scalar_type_columnar_roundtrip(scalar_type: ScalarType) {
-        use mz_persist_types::columnar::validate_roundtrip;
+    #[track_caller]
+    fn scalar_type_parquet_roundtrip<'a>(
+        scalar_type: ScalarType,
+        datums: impl IntoIterator<Item = Datum<'a>>,
+    ) {
+        use mz_persist_types::parquet::validate_roundtrip;
```

Comment:
Using the parquet encoding here is what caught the bug in `arrow2` that motivated the switch to `arrow-rs`, previously we were only round-tripping through a `Part` which doesn't exercise structured Arrow or Parquet